public/index.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
```

Comment:
Why do we need this file? Webpack dynamically generates `public/index.html`.

Reply:
we don't folder should be in .gitignore, i'll add it there

Reply:
i think i already added it, but you'll have to `git rm ./public/index.html`